apis/Google.Cloud.Compute.V1/Google.Cloud.Compute.V1.IntegrationTests/ComputeFixture.cs

```diff
@@ -27,51 +27,70 @@ public class ComputeFixture : CloudProjectFixtureBase, ICollectionFixture<Comput
         // avoid requiring another one.
         private const string TestRegionEnvironmentVariable = "TEST_PROJECT_LOCATION";
 
-        /// <summary>
-        /// Region to use for testing.
-        /// </summary>
-        public string Region { get; }
-
-        /// <summary>
-        /// The prefix to use for resources generated by this test fixture, so they can be cleaned up.
-        /// </summary>
-        public string ResourcePrefix { get; }
-
         public ComputeFixture()
         {
             Region = Environment.GetEnvironmentVariable(TestRegionEnvironmentVariable);
+            Zone = "us-central1-a";
             if (string.IsNullOrEmpty(Region))
             {
-                throw new InvalidOperationException($"Environment variable {TestRegionEnvironmentVariable} must be set");
+                throw new InvalidOperationException(
+                    $"Environment variable {TestRegionEnvironmentVariable} must be set");
             }
-            ResourcePrefix = IdGenerator.FromDateTime(prefix: "test-", suffix: "-");
+
+            ResourcePrefix = IdGenerator.FromDateTime("test-", "-");
         }
 
-        public string GenerateResourceId() => IdGenerator.FromGuid(prefix: ResourcePrefix);
+        /// <summary>
```

Comment:
Why have you moved these? We generally keep properties before constructors. If this is Resharper reformatting, please just revert it all - it's made a lot of changes, almost all of which have negative results on readability.

Reply:
These were moved by resharper, I haven't found editorconfig file in this repo so I used the one from dotnet/corefx repo. 
I reverted all the changes made by resharper and fixed your comments manually